src/popup/components/AppSettings.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="content">
+    <div class="p-3 container">
+      <p class="font-weight-bold align-self-center">App Settings</p>
+
+      <table class="settings-list">
+        <tr>
+          <td>
+            <toggle-button
+              :value="showSLButton"
+              :sync="true"
+              color="#b02a8f"
+              :width="30"
+              :height="18"
+              @change="handleToggleSLButton()"
+            />
+          </td>
+          <td>
+            Show SimpleLogin button on email input fields<br />
+            <small>
+              SimpleLogin button allows you to quickly create random alias
```

Comment:
can we change the text to 
"If enabled, you can quickly create a random alias by clicking on the SimpleLogin button placed next to the email field" ?